src/provisioner/provisioner.py

```diff
@@ -84,17 +112,17 @@ def config_apply(solution_conf_url: str, cortx_conf_url: str = None, \
                     val = Cipher.encrypt(cipher_key, val)
                     # decoding the byte string in val variable
                     Conf.set(CortxProvisioner._solution_index, key, val.decode('utf-8'))
-            CortxProvisioner.config_apply_cortx(cortx_conf)
+            CortxProvisioner.solution_config_apply(cortx_conf, cortx_release)
 
     @staticmethod
-    def config_apply_cortx(cortx_conf):
+    def solution_config_apply(cortx_conf, cortx_release):
         """ Convert CORTX config into confstore keys """
-        cortx_config = Conf.get(CortxProvisioner._solution_index, 'cortx')
-        cs = CortxConfig(cortx_config)
-        cs.save(cortx_conf)
+        config_info = Conf.get(CortxProvisioner._solution_index, 'cortx')
+        cortx_config = CortxConfig(config_info, cortx_release)
+        cortx_config.save(cortx_conf)
 
     @staticmethod
-    def config_apply_cluster(cortx_conf):
+    def solution_cluster_apply(cortx_conf, cortx_release):
```

Comment:
Can we rename this instead to the actual action i.e. "apply_cluster_config" because we do apply cluster conf.

Reply:
Done.